Automattic/wp-calypso
# Staging Sites: add staging site to hosting configuration submenu

- Close https://github.com/Automattic/dotcom-forge/issues/1909

## Proposed Changes

* This PR adds **Staging site** item to the Hosting Configuration submenu on SMP

## Testing Instructions

*  Create a staging site from **Settings > Hosting Configuration** of an Atomic site
*  Go to `/sites`
* Click on the ellipsis menu by an Atomic site:

<img width="1208" alt="Screenshot 2023-03-09 at 1 40 54 PM" src="https://user-images.githubusercontent.com/25575134/224026029-3f722250-4a06-4bcf-b29f-3929fe07c847.png">

* Click on **Hosting Configuration** > **Staging site**:

<img width="1385" alt="Screenshot 2023-03-09 at 1 42 03 PM" src="https://user-images.githubusercontent.com/25575134/224026362-90741529-52cd-4a93-84d0-8cf523e8158d.png">

* Check if it brings you to the **staging card** under `hosting-config`

**Testing instructions for a staging site**

*  Create a staging site from **Settings > Hosting Configuration** of an Atomic site
*  Go to `/sites`
* Click on the ellipsis menu by a staging site site
* Click **Hosting Configuration**. The **Staging site** submenu item should not be available

<img width="1333" alt="Screenshot 2023-03-09 at 1 45 43 PM" src="https://user-images.githubusercontent.com/25575134/224027054-80423bfe-f81e-419f-9514-b827efba0197.png">

## Pre-merge Checklist
